ffmpeg/lpms_ffmpeg.c

```diff
@@ -115,8 +115,9 @@ struct filter_ctx {
   //      ( pts received from filter will be wrong by however many flushed
```

Comment:
Is this comment block still accurate given the new timestamp handling approach?

Reply:
Good catch, completely ignored the comments in this struct while making changes. Will remove thx.